docs/proposals/multiple-volumegroups.md

```diff
@@ -190,7 +190,7 @@ This proposal would insert `capacity.topolvm.cybozu.com/<device-class>` as follo
 ```yaml
 spec:
   containers:
-  - name: testhttpd
+  - name: pause
```

Comment:
IMO we don't need to edit this file since it is a proposal, which need not to be kept up-to-date.

Reply:
Thank you. I fixed it.